lib/modules/platform/bitbucket/index.ts

```diff
@@ -309,6 +310,33 @@ export async function findPr({
   if (pr) {
     logger.debug(`Found PR #${pr.number}`);
   }
+
+  /**
+   * Bitbucket doesn't support renaming or reopening declined PRs.
+   * Instead, we have to use comment-driven signals.
+   */
+  if (pr && pr.state === 'closed') {
+    const REOPEN_PR_COMMENT_KEYWORD = 'reopen!';
+
+    const reviewers = await effectiveDefaultReviewers(config.repository);
+    const reviewersUuid = reviewers.map((reviewer) => reviewer.uuid);
```

Comment:
use a set here? 🤔

Reply:
I've renamed the variable to be clearer of it's intent.  It's an array of reviewer uuids